### PR TITLE
Update nio.env and nio.conf for SSL and nio_lite binaries

### PR DESCRIPTION
**Pull request by gingajake**

### CODE REVIEW

File: nio.conf [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
 # discoverable components that should be disabled from project
 #
-#disabled=SNMPAgent, PubkeeperServer
+#disabled=SNMPAgent
```
Comment (mattdodge) [inline nio.conf]: Is this change intended to be here? We should just disable both components (aka uncomment the line) rather than have two "disabled" lines in the example conf

File: nio.env [hunk 2]
```diff
@@ -11,22 +11,22 @@ INSTANCE_NAME: local
 INSTANCE_ID: local
 
 # Pubkeeper Client
-PK_HOST: 127.0.0.1
-PK_PORT: 9898
-PK_TOKEN: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwZXJtIjpbImFsbCJdLCJpc3MiOiJwdWJrZWVwZXIiLCJhdWQiOiJwdWJrZWVwZXIiLCJzdWIiOiJndWVzdCJ9.ODsw4iGEm4OeW10mqHEuMxqVOOZtITnETopee5GKb7M
-PK_SECURE: False
+PK_HOST:
```
Comment (mattdodge) [inline nio.env]: can we make the default `your-system-id.pubkeeper.nio.works`, same for WS_HOST

**Review by mattdodge — CHANGES_REQUESTED** [review]
Couple things, also we should test this (if we haven't already) with a brand new environment with the current binary to make sure the right things are installed

File: nio.conf (hunk 1 above)
Comment (gingajake) [reply nio.conf]: Yes this change was intended, wasn't positive if we needed the SNMP agent so I didn't disable.  Since we're setting up for a nio_lite binary, figured we'd have the PubkeeperServer disabled by default.

File: nio.env (hunk 2 above)
Comment (gingajake) [reply nio.env]: Should it be your-system-id.pubkeeper.nio.works or your-system-id.pubkeeper.nio.solutions?

**Review by mattdodge — APPROVED** [review]
